liquibase-core/src/main/java/liquibase/change/core/AddNotNullConstraintChange.java

```diff
@@ -95,24 +96,20 @@ public void setConstraintName(String constraintName) {
 
     @Override
     public SqlStatement[] generateStatements(Database database) {
-
```

Comment:
Don't remove the commented out sqlite support

Reply:
I re-added it.